PdCore/src/main/jni/Application.mk

```diff
@@ -1,3 +1,3 @@
-APP_PLATFORM := android-17
+APP_PLATFORM := android-28
```

Comment:
This PR raises the minimum supported Android version of pd-for-android to 28. This is done because libpd now depends on the C 'glob' function which is natively supported on Android first from API 28.